client/src/frontend/components/composed/EntityHero/EntityHero.js

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect, useContext, useRef } from "react";
+import PropTypes from "prop-types";
+import { FrontendModeContext } from "helpers/contexts";
+import * as Styled from "./styles";
+
+export default function EntityHero({
+  TitleComponent,
+  TopLeftComponent,
+  TopRightComponent,
+  BottomLeftComponent,
+  BottomRightComponent,
+  ImageComponent,
+  lightMode = false,
+  theme = "project"
+}) {
+  const resizeId = useRef(null);
+  const rightColRef = useRef();
+  const titleRef = useRef();
+  const { isStandalone } = useContext(FrontendModeContext);
+
+  const addStandaloneMargin = () => {
+    if (!rightColRef.current || !titleRef.current) return;
+    const height = titleRef.current.offsetHeight;
+    rightColRef.current.style.marginTop = `${height}px`;
+  };
+
+  useEffect(() => {
+    const handleResize = () => {
+      if (resizeId.current) {
+        window.cancelAnimationFrame(resizeId);
+      }
+      resizeId.current = window.requestAnimationFrame(() => {
+        addStandaloneMargin();
+      });
+    };
+
+    if (isStandalone) {
+      window.addEventListener("resize", handleResize);
+      return window.removeEventListener("resize", handleResize);
+    }
+  }, [isStandalone]);
```

Comment:
Making a note that we should ask if journals can be in standalone mode, and if not to move this logic to the project pattern.